src/services/id-service.api-service.ts

```diff
@@ -36,4 +39,34 @@ export class IdServiceApiService {
         }
       });
   }
+
+  public async getByUri<T>(uri: string, getSso: () => Promise<string>): Promise<IHttpResponse<T>> {
+    try {
+      const sso = await getSso();
+      const response = await this.http.createRequest(`${oeAppConfig.idServiceUrl + '/uris?uri=' + uri}`)
+      .asGet()
+      .withHeader('Authorization', 'Bearer ' + sso)
```

Comment:
is deze er altijd, of moeten we die optioneel maken?

Reply:
Voor zover ik al heb gezien in de applicaties gaan we die altijd moeten meesturen